lib/embed/embed.dart

```diff
@@ -96,6 +96,25 @@ class PlaygroundMobile {
     });
   }
 
+  /**
+   * Return if strong mode should be enabled
+   */
+  bool _isStrongMode(String strong) {
```

Comment:
Passing to analysis is handled in playground.dart [here](https://github.com/dart-lang/dart-pad/blob/ea0a7492751e2ba94ef496a1b616b167f46b7a9f/lib/playground.dart#L643)